src/context.h

```diff
@@ -8,7 +8,7 @@
 #include "core/streaming.h"
 
 #include <vector>
-#include <nlohmann/json_fwd.hpp>
+#include <rsutils/json.h>
```

Comment:
Should be <nlohmann/json_fwd.hpp>, I think

Reply:
Updated to <nlohmann/json.hpp>